bank2ynab.conf

```diff
@@ -340,12 +351,23 @@ Input Columns = skip,Date,skip,skip,Memo,skip,skip,skip,skip,skip,skip,Payee,ski
 # Support for all banks with a solution produced by Bankernes EDB Central
 # Tested Banks: Arbejdernes Landsbank, Merkur Andelskasse, DK Handelsbanken,
 # add more here..
-Source Filename Pattern = Posteringer
+Source Filename Pattern = Al
 Source CSV Delimiter = ;
 Header Rows = 0
 Input Columns = Date,Payee,Inflow,Running Balance
 Date Format = %d-%m-%Y
 
+[DK Portalbank]
```

Comment:
This is generating a conflict  - can you remove and pull from `develop` please?

Reply:
Ye the whole PR would need a rebase on a recent origin/develop to work. :+1: